modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -580,7 +590,8 @@ export default {
             active: true
           });
           setTimeout(async () => {
-            this.modal.showModal = false;
+            this.filters.archived = false;
+            this.loadDoc();
```

Comment:
Hang on... so you mark it busy, then you wait a full second before doing anything, then you load the document but you don't await it, and then you immediately mark it not busy. Does that make sense?

Reply:
No, some of that is a mistake, but also it's complicating the UX. Got rid of the timeout and just re populate the doc with the freshly published one.